api/src/main/java/io/opentelemetry/trace/Tracestate.java

```diff
@@ -45,6 +45,18 @@
   private static final int VALUE_MAX_SIZE = 256;
   private static final int MAX_KEY_VALUE_PAIRS = 32;
 
+  /**
+   * Returns the default {@code Tracestate} with no entries.
+   *
+   * @return the default {@code Tracestate}.
+   * @since 0.1.0
+   */
+  public static Tracestate getDefault() {
+    return DEFAULT;
+  }
+
+  private static final Tracestate DEFAULT = Tracestate.builder().build();
```

Comment:
nit: Consider to move this before the method (top of the class.)

Reply:
Done for all classes.